Digipost.Signature.Api.Client.Docs/Direct/DirectClientExample.cs

```diff
@@ -44,6 +45,50 @@ public async Task CreateAndSendSignatureJob()
             var directJobResponse = await directClient.Create(job);
         }
 
+        public async Task RequestNewRedirectUrlFromResponse()
+        {
+            ClientConfiguration clientConfiguration = null; //As initialized earlier
+            Job job = null; //As created earlier
+            var directClient = new DirectClient(clientConfiguration);
+            var directJobResponse = await directClient.Create(job);
+
+            var signerFromResponse = directJobResponse
+                .Signers
+                .First(s => s
+                    .Identifier
+                    .IsSameAs(new PersonalIdentificationNumber("12345678910"))
+                );
+
+            var newRedirectUrl = await directClient.RequestNewRedirectUrl(signerFromResponse);
+        }
+
+        public async Task RequestNewRedirectUrlFromSignerUrl()
+        {
+            ClientConfiguration clientConfiguration = null; //As initialized earlier
+            Job job = null; //As created earlier
+            var directClient = new DirectClient(clientConfiguration);
+            var directJobResponse = await directClient.Create(job);
+
+            // Step 1:
+            foreach (var signerResponse in directJobResponse.Signers)
+            {
+                //Persist signer URL in sender system
+                var signerResponseSignerUrl = signerResponse.SignerUrl;
+            }
+
+            // ... some time later ...
+
+            // Step 2: Request new redirect URL for signer
+            Uri persistedSignerUrl = null; //Persisted URL from step 1.
+            var signerWithUpdatedRedirectUrl = await directClient
+                .RequestNewRedirectUrl(
+                    NewRedirectUrlRequest
+                        .FromSignerUrl(persistedSignerUrl)
+                );
+
+            var newRedirectUrl2 = await directClient.RequestNewRedirectUrl(signerWithUpdatedRedirectUrl);
```

Comment:
Minor detail, men noe sånt blir vel mest riktig?
```suggestion
            var newRedirectUrl = await directClient.RequestNewRedirectUrl(signerWithUpdatedRedirectUrl).RedirectUrl;
```

Reply:
Eller... trenger du denne linjen i det hele tatt? Du har jo fått ny redirect-url i `signerWithUpdatedRedirectUrl`. Dette må jo være hvis du _enda en gang_ vil ha ny redirect-url.

Reply:
Jupp!